server/domain/student/models/Student.ts

```
@@ -0,0 +1,161 @@
+  }
+
+  getTargetGpa(): number | null {
+    return this.targetGpa;
+  }
+
+  getAcademicInfo(): AcademicInfo {
+    return this.academic;
+  }
+
+  equals(other: Student): boolean {
+    if (!(other instanceof Student)) return false;
```

ESLint

[error] Expected { after 'if' condition.

(curly)